doc/build-unix.md

```diff
@@ -139,6 +139,46 @@ The GUI will be able to encode addresses in QR codes unless this feature is expl
 
 Otherwise, if you don't need QR encoding support, use the `-DWITH_QRENCODE=OFF` option to disable this feature in order to compile the GUI.
 
+### Alpine
+
+#### Dependency Build Instructions
+
+Build requirements:
+
+    apk add build-base cmake libressl libtool linux-headers python3
```

Comment:
`libressl` ?

Reply:
Crikey that's an old hangover....

I ported initial requirements from my current [Dockerfile](https://github.com/willcl-ark/bitcoin-core-docker/blob/d3676df64a7e5c4b816dc5a6ca542098e5d498f8/master/alpine/Dockerfile#L15)

I foolishly assumed this was just the case of Alpine splitting packages more granularly that e.g. Ubuntu, but taking a closer look now suspect this is a hangover from pre 0.12 where we supported SSL on RPC?

Removed in 6b6819e75f6371b3dbfcc3ba8379379b6637fe31